CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## unreleased
+
+* Updated `css-loader` to `^7.1.2` and changed `css-loader` options to restore 6.x behavior.
+* Updated the minimum version of Node to `^18.12.0`.
```

Comment:
I guess we didn't update the change log when we change the node version. ;(

```suggestion
* Updated the minimum version of Node to `^18.12.0`.
```

Reply:
minimum node version was updated in #345 but I missed the changelog, so I added in this time...

Reply:
```suggestion
* Updated the minimum version of Node to `18.12.0`.
```